node/snapshots.re

```diff
@@ -11,16 +11,19 @@ type t = {
   last_block: Block.t,
   last_block_signatures: Signatures.t,
   additional_blocks: list(Block.t),
+  last_seen_membership_change_timestamp: float,
```

Comment:
Why was this added to the snapshots?

Reply:
So that all the validators have a common notion of `last_seen_membership_change_timestamp`

Reply:
As explained offline, since this property is local consensus, ie values other nodes need not agree, I will be removing them from the snapshots.